### DISCUSSION TITLE
Prioritizing PRs for next CTP and merge.

### DISCUSSION THREAD

**Discussion by singhashish-wpf** [dotnet/wpf#9920] [General]
As we continue to improve the WPF platform, we're looking to gather your insights and feedback on which pull requests (PRs) should be prioritized for the next test pass and considered for merging. As of now we are prioritizing PRs that address performance and bug fixes.

Please comment the PR links, upvote if the link is already commented by someone else. 

**Notes:**

- All these PRs will be merged into main, hence this is for .NET 10.
- PR merge is dependent upon a green result from the test pass. This process gets complex when we take in batch PRs and one of them creates a failure which needs further debugging and fixing the test or issue and then retesting. 
- PR merge is also dependent upon @dotnet/dotnet-wpf-triage group approving the PRs.

**Comment by lindexi** [comment]
I recommend @h3xds1nz 's PRs

https://github.com/dotnet/wpf/issues?q=is%3Apr+is%3Aopen+author%3Ah3xds1nz

**Reply by h3xds1nz** [reply]
Thanks @lindexi. I'll put the updated priority list here later today from my side as I still have some blockers.

**Comment by lindexi** [comment]
- https://github.com/dotnet/wpf/pull/4425

**Comment by lindexi** [comment]
- https://github.com/dotnet/wpf/pull/6840

**Comment by lindexi** [comment]
- https://github.com/dotnet/wpf/pull/9463

**Reply by h3xds1nz** [reply]
I think this is better addressed in #9468, I was originally hoping yours might get to .NET 9 to fix the regression itself as its a very simple change and then replaced with my more complex overhaul.

**Comment by h3xds1nz** [comment]
From my side in the next CTP I'd like to see #9683 and #9697 as those are direct blockers for MouseGesture/KeyGesture and the whole finish on this side of converters.

Then #9822 and #9739 could be bundled together to allow for further big optimizations I work on (where I'm also currently blocked).

This #4027 from @lindexi should either be taken or closed but I'd take it as it improves the code quality. It is so simple and makes sense to take it imho. Shouldn't be there for almost 4 years without a decision.

This PR #7485 from @ThomasGoulet73, I literally almost made the same last week if I didn't search first, I do not see a single reason why this is still there on .NET Core. It has passed tests afaik previously and it is just waiting for decision (unfortunately for almost 2 years). It is a pure dead code.

And last thing, I'd like to span out #9468 #9361 and #9352 over different CTPs. They touch the same thing but each on different places and I think it's good to make it separate if I made a mistake somewhere, it would be easier to figure out if they're taken over different runs.

**Comment by h3xds1nz** [comment]
#9902 And I'd like this @ThomasGoulet73 PR to be considered as well, I really appreciate him doing the conversions from CLI to C#, he's making it as 1:1 as possible for easy reviews. I believe he will be able to invest more when the PRs are moving faster on managed dwrite.

**Comment by franchyd** [comment]
https://github.com/dotnet/wpf/pull/6606 has been stagnant for a while now.

**Comment by h3xds1nz** [comment]
@pchaurasia14 @singhashish-wpf It's pronnounced "hexed" and "sins", haha.

Happy to see you got two sessions in on .NET conf! Now, if that's not much to ask, please share with us when we're getting new PR merges in since a lot of them now linger in "Status:Completed" after tests ran by @harshit7962, since for myself I'm pretty much blocked these days.

**Reply by lindexi** [reply]
Haha, happy to see on .NET conf.

**Reply by pchaurasia14** [reply]
@h3xds1nz  - Thanks for the clarification :) The new PRs should be in by next week.
